lib/ansiblelint/formatters/__init__.py

```diff
@@ -26,3 +26,22 @@ def format(self, match):
                                 "E" + match.rule.id,
                                 match.message,
                                 )
+
+
+try:
+    from ansible.color import stringc
+    ANSIBLE_VERSION = 1
+except ImportError:
+    from ansible.utils.color import stringc
+    ANSIBLE_VERSION = 2
```

Comment:
don't really need this in this file (it's needed in utils.py due to checks in other methods)